src/arch/xtensa/lib/cpu.c

```diff
@@ -46,7 +46,7 @@ static void alloc_shared_secondary_cores_objects(void)
 {
 	uint8_t *dynamic_vectors;
 
-	dynamic_vectors = rzalloc(SOF_MEM_ZONE_RUNTIME_SHARED, 0, 0, SOF_DYNAMIC_VECTORS_SIZE);
+	dynamic_vectors = rzalloc(0, 0, SOF_DYNAMIC_VECTORS_SIZE);
```

Comment:
this is allocating uncached memory, so with zones removed we now have to use the `SOF_MEM_FLAG_COHERENT` flag